changelogs/archive_fragments/2.0.0/2.0.0.yaml

```diff
@@ -0,0 +1,2 @@
+release_summary: |
+  This release provides Dell SONiC Enterprise Ansible Collection support for SONiC 4.x images. It is the first release for the 2.x branch of the collection. Subsequent enhancements for support of SONiC 4.x images will also be provided as needed on the 2.x branch. This release also contains bugfixes and enhancements to supplement the Ansible functionality provided previously for SONiC 3.x images. The changelog describes changes made to the modules and plugins included in this collection since release 1.1.0.
```

Comment:
It might look better if these long lines are broken into multiple lines. Same for the subsequent `.yaml` changelog files too.

Reply:
It's a yaml file input to the changelog generator. I'm not sure if a multi-line "value" will work. 

The format specification doesn't show any cases of multi-line values, but I can try it. (I'm not sure I can re-run the changelog generator in this workspace, but, again, I can try it and see what happens. I may need to start over with a new workspace.)

Reply:
One other note: This is just an input to the changelog generator. The "real" rendering is the CHANGELOG.rst file.